phpseclib/Net/SFTP.php

```diff
@@ -829,11 +805,9 @@ private function nlist_helper($dir, $recursive, $relativeDir)
     /**
      * Returns a detailed list of files in the given directory
      *
-     * @param string $dir
-     * @param bool $recursive
      * @return array|false
      */
-    public function rawlist($dir = '.', $recursive = false)
+    public function rawlist(string $dir = '.', bool $recursive = false)
```

Comment:
Prob ought to update these to return null on failure vs false. Kinda like what I proposed with `PublicKeyLoader::load`. Can be done with a later PR however.